### ISSUE TITLE
Query errors are silently ignored

### ISSUE THREAD

**Issue by josh-newman** [Tinder/bazel-diff#81]
Bazel queries are executed by [starting a process and reading the output](https://github.com/Tinder/bazel-diff/blob/bd6d9022ef747a101866159b485ce1c35f69c081/src/main/java/com/bazel_diff/BazelClient.java#L105) but existing code doesn't check for bazel errors. When I ran this in a workspace with some repository rule errors, bazel-diff exited with status 0 (and no error messages) after printing partial output.

I discovered my error by adding `pb.redirectError(ProcessBuilder.Redirect.INHERIT)`. However, maybe you'd want to do something a little more sophisticated (like check bazel exit status and only print stderr if there's a problem), and also set bazel-diff's error code.

(Also, thanks for sharing this tool publicly!)

**Comment by tinder-maxwellelliott** [comment]
Hello there @josh-newman , you can disable this via this flag

```terminal
  -k, --[no-]keep_going   This flag controls if `bazel query` will be executed
                            with the `--keep_going` flag or not. Disabling this
                            flag allows you to catch configuration issues in
                            your Bazel graph, but may not work for some Bazel
                            setups. Defaults to `true`
```

I have make true the default for the existing users, in the future this will flip

**Comment by josh-newman** [comment]
Wouldn't you still need to check `process.exitValue()` to know there was a bazel error?

**Comment by tinder-maxwellelliott** [comment]
Let me put together a PR to make sure we are on the same page

**Comment by tinder-maxwellelliott** [comment]
Fixed via #89

**Comment by josh-newman** [comment]
Thanks! (Sorry for the delay, I got pulled away to some other things.)
